arrocke/gloss-translation
# feat: implement data import UI

## What has changed



## Connected Issues



Related to #60

## QA Steps



1. Go to the manage page for a language
1. Click the import button
1. The backend will simulate 10 seconds of processing. The spinning indicator by the import button should spin while the processing occurs.
1. A success flash should be flashed

## Post-Deployment
